.github/workflows/CompatHelper.yml

```diff
@@ -1,7 +1,7 @@
 name: CompatHelper
 on:
   schedule:
-    - cron: 0 0 * * *
```

Comment:
@Xiaoyan-Li This changes CompatHelper to run only monthly, on the first of each month, to reduce the spamminess of the pull requests.